ga4gh/cli.py

```diff
@@ -109,11 +109,24 @@ def _run(self, methodName, attrName=None):
         method = getattr(self._httpClient, methodName)
         results = method(self._request)
         for result in results:
-            if attrName is None:
-                print(result)
+            if self._verbosity == 1:
+                print(methodName, "request: \n", self._request)
+                print(methodName, "response: \n", result)
+            elif self._verbosity > 1:
+                requestJson = json.dumps(
+                    json.loads(self._request.toJSONString()),
+                    sort_keys=True, indent=4)
+                print(methodName, "request json string: \n", requestJson)
+                responseJson = json.dumps(
```

Comment:
I think these dump/load calls are probably redundant if you have access to the string to begin with... why can't you just print `response.toJSONString()`?